rdrf/rdrf/models/definition/models.py

```diff
@@ -55,7 +55,9 @@ def get_by_natural_key(self, code):
         return self.get(code=code)
 
     def get_by_comma_separated_codes(self, codes):
-        return self.filter(code__in=[s.strip() for s in codes.split(",")])
```

Comment:
What's the significance of this change? I don't understand what it's for.

Returning multiple instances of a CDE if it appears in `codes` twice? Returning a list rather than a QuerySet?

Reply:
It is for keeping the sections in the same order as the codes are in the comma separated list.

Reply:
Ah! Obviously 😅